stl/inc/span

```diff
@@ -0,0 +1,723 @@
+        return iterator(data(), size(), 0);
+#else // ^^^ _ITERATOR_DEBUG_LEVEL >= 1 ^^^ // vvv _ITERATOR_DEBUG_LEVEL == 0 vvv
+        return iterator(data());
+#endif // _ITERATOR_DEBUG_LEVEL
+    }
+
+    _NODISCARD constexpr iterator end() noexcept {
+#if _ITERATOR_DEBUG_LEVEL >= 1
+        return iterator(data(), size(), size());
+#else // ^^^ _ITERATOR_DEBUG_LEVEL >= 1 ^^^ // vvv _ITERATOR_DEBUG_LEVEL == 0 vvv
+        return iterator(data() + size());
+#endif // _ITERATOR_DEBUG_LEVEL
+    }
+
+    _NODISCARD constexpr const_iterator begin() const noexcept {
+#if _ITERATOR_DEBUG_LEVEL >= 1
+        return const_iterator(data(), size(), 0);
+#else // ^^^ _ITERATOR_DEBUG_LEVEL >= 1 ^^^ // vvv _ITERATOR_DEBUG_LEVEL == 0 vvv
+        return const_iterator(data());
+#endif // _ITERATOR_DEBUG_LEVEL
+    }
+
+    _NODISCARD constexpr const_iterator end() const noexcept {
+#if _ITERATOR_DEBUG_LEVEL >= 1
+        return const_iterator(data(), size(), size());
+#else // ^^^ _ITERATOR_DEBUG_LEVEL >= 1 ^^^ // vvv _ITERATOR_DEBUG_LEVEL == 0 vvv
+        return const_iterator(data() + size());
+#endif // _ITERATOR_DEBUG_LEVEL
+    }
+
+#ifdef __cpp_lib_concepts
+    _NODISCARD friend constexpr const_iterator begin(span _Right) noexcept {
+        // non-member overload to model the exposition-only forwarding-range concept
+        return _Right.begin();
+    }
+    _NODISCARD friend constexpr const_iterator end(span _Right) noexcept {
+        // non-member overload to model the exposition-only forwarding-range concept
+        return _Right.end();
+    }
+#endif // __cpp_lib_concepts
+
+    _NODISCARD constexpr const_iterator cbegin() const noexcept {
+        return begin();
+    }
+
+    _NODISCARD constexpr const_iterator cend() const noexcept {
+        return end();
+    }
+
+    _NODISCARD constexpr reverse_iterator rbegin() noexcept {
+        return reverse_iterator{end()};
+    }
+
+    _NODISCARD constexpr reverse_iterator rend() noexcept {
+        return reverse_iterator{begin()};
+    }
+
+    _NODISCARD constexpr const_reverse_iterator rbegin() const noexcept {
+        return const_reverse_iterator{end()};
+    }
+
+    _NODISCARD constexpr const_reverse_iterator rend() const noexcept {
+        return const_reverse_iterator{begin()};
+    }
+
+    _NODISCARD constexpr const_reverse_iterator crbegin() const noexcept {
+        return rbegin();
+    }
+
+    _NODISCARD constexpr const_reverse_iterator crend() const noexcept {
+        return rend();
+    }
+
+    _NODISCARD constexpr pointer _Unchecked_begin() noexcept {
+        return data();
+    }
+
+    _NODISCARD constexpr const_pointer _Unchecked_begin() const noexcept {
+        return data();
+    }
+
+    _NODISCARD constexpr pointer _Unchecked_end() noexcept {
+        return data() + size();
+    }
+
+    _NODISCARD constexpr const_pointer _Unchecked_end() const noexcept {
+        return data() + size();
+    }
+
+    constexpr void swap(span& _Other) noexcept {

```

Comment:
`span` has no member `swap` in the working draft.

Reply:
I have removed it, although is there a reason not to have a swap implemented?